### PR TITLE
[EN] Polished Game_Modifers

### PR DESCRIPTION
**Pull request by ElectrifyPro**
### Description

Improved the quality of text in Game_Modifers, fixed grammar issues, and made some unclear text more understandable.
**Did not** change text written in British English.

### Status

Finished

### Changelog

Not sure what to put here..

### CODE REVIEW

File: wiki/Game_Modifiers/en.md [hunk 1]
```diff
@@ -78,9 +78,9 @@
 
 Game modifiers (mods) are optional modifications to a beatmap's elements and settings that the player can enable at will. A mod effect can range from simple gameplay changes like changing the beatmap's settings to automation such as allowing _osu!_ to do a perfect replay (Auto), on a condition of possible Score Multiplier changes to making the play unrankable. When multiple mods are enabled, the Score Multiplier is the product of the enabled mods' Score Multiplier. For example, when Hidden and Flashlight are applied at the same time, the Score Multiplier will be 1.19x (1.06 * 1.12 = 1.1872; 1.19 by rounding up to nearest hundredths).
 
-Shortcut keys to enable the mods can be changed in the Options under the Keyboard header inside `Change keyboard bindings` with a header called "Mod Select". The usage of the shortcut keys will only work when the mods screen is shown.
+Shortcut keys to enable the mods can be changed in the Options under the Keyboard header inside `Change keyboard bindings` with a header called "Mod Select". The usage of the shortcut keys will only work when the mod selection screen is shown.
```
Comment (this1neguy) [inline wiki/Game_Modifiers/en.md]: * remove 'the' before mods
* 'in the options under the keyboard inside...' is a whole lot of prepositional phrases, suggest rewording this

File: wiki/Game_Modifiers/en.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ Do note that the usage of No Fail mod reduces the amount of possible [Performanc
 
 > Shortcut key: `E`
 
-> Less zoom. ("Zoom" meaning here means how fast the beatmap's BPM is)
+> Less zoom. ("Zoom" referring to how fast the beatmap's BPM is)
```
Comment (this1neguy) [inline wiki/Game_Modifiers/en.md]: * "'Zoom' referring to the beatmap's BPM"? i don't feel this needs the 'fast' modifier

File: wiki/Game_Modifiers/en.md [hunk 3]
```diff
@@ -228,25 +228,25 @@ Using this mod on osu!mania will set the play as unranked play.
...
+Click on Sudden Death again to activate Perfect mod.
 
 > Score Multiplier: 1.00x
 
 > Shortcut key: `SS` (from unselected) or `Shift`+`S`
 
 > SS or quit.
 
-Perfect mod causes _any_ imperfect hit (as in, any score values that does not give 100% accuracy) to be counted as a **miss** and the song will **forcefully restart automatically**; something Sudden Death does not do.
+Perfect mod causes _any_ imperfect hit (as in, any score values that do not give 100% accuracy) to be counted as a **miss** and the song will **forcefully restart automatically**; something Sudden Death does not do.
```
Comment (this1neguy) [inline wiki/Game_Modifiers/en.md]: * 'any score value that does not..." to agree with 'hit' prior to the parenthetical phrase
* 'and forcefully restarts the song automatically, something..."
** don't need the semicolon and this fits the verb tenses

File: wiki/Game_Modifiers/en.md [hunk 4]
```diff
@@ -228,25 +228,25 @@ Using this mod on osu!mania will set the play as unranked play.
...
 
-While this mod gives no Score Multiplier, it is _more unforgiving_ (100% accuracy or force auto-restart until the player give up by quitting the beatmap manually) when compared to Sudden Death (as long as the combo did not break). There will not be any failed screen and if the beatmap has a skipable prologue, it will be played _again_ since Perfect mod uses restart functionality (not quick-restart) the moment a miss (by Perfect's standard) is given.
+While this mod gives no Score Multiplier, it is _more unforgiving_ (100% accuracy or force auto-restart until the player gives up by quitting the beatmap manually) when compared to Sudden Death (as long as the combo did not break). There will not be any failed screen and if the beatmap has a skipable prologue, it will be played _again_ since Perfect mod uses restart functionality (not quick-restart) the moment a miss (by Perfect's standard) is given.
```
Comment (this1neguy) [inline wiki/Game_Modifiers/en.md]: * skipable -> skippable

File: wiki/Game_Modifiers/en.md [hunk 5]
```diff
@@ -270,15 +270,15 @@ The method used to increase the speed does not increase the pitch of the song, b
 
 ---
 
-In [osu!taiko][osu!taiko wikilink] mode, since the hit leniency is greatly increased on top of osu!taiko's already strict overall difficulty in addition to making slider ticks a lot harder to hit and reducing the value of dendens, Double Time is usually regarded as the hardest mod in osu!taiko and is seldom used.
+In [osu!taiko][osu!taiko wikilink] mode, since the hit leniency is greatly reduced on top of osu!taiko's already strict overall difficulty in addition to making slider ticks a lot harder to hit and reducing the value of dendens, Double Time is usually regarded as the hardest mod in osu!taiko and is seldom used.
```
Comment (this1neguy) [inline wiki/Game_Modifiers/en.md]: * remove "a lot", if emphasis is needed recommend "much" or even "considerably" in favor of the less professional-sounding "a lot"
* maybe put "in addition to ... dendens" in parentheses? this sentence is a massive run-on atm

File: wiki/Game_Modifiers/en.md [hunk 6]
```diff
@@ -336,17 +336,17 @@ In [osu!catch][osu!catch wikilink] mode, Hidden causes the fruits to fade out ab
...
+Click on Fade In again to activate Hidden mod.
 
 > Shortcut key: `FF` (from unselected) or `Shift`+`F` (osu!mania only)
 
 > The notes fade out before you hit them!
 
 ![Hidden play in osu!mania mode (Left: Normal; Right: DDR style)][HD_M]
 
-In [osu!mania][osu!mania wikilink] mode, Hidden can be used by pressing Fade In mod again. Inverse of Fade In behaviour.
+In [osu!mania][osu!mania wikilink] mode, Hidden can be used by pressing Fade In mod again, which functions as the inverse of Fade In.
```
Comment (this1neguy) [inline wiki/Game_Modifiers/en.md]: * replace your change with "and functions as" for brevity

File: wiki/Game_Modifiers/en.md [hunk 7]
```diff
@@ -630,7 +630,7 @@ A detailed explanation about the second playfield can be found in the [osu!mania
 
 ![Comparison between original (left), Random only (middle), and Random with Co-Op (right) for osu!mania-specific (top) and non-specific (bottom) beatmap.][Random1]
 
-Random mod will **change the note placement** only.
+Random mod will **randomizes the note placement** only.
```
Comment (this1neguy) [inline wiki/Game_Modifiers/en.md]: * 'randomize' not 'randomizes'

File: wiki/Game_Modifiers/en.md [hunk 8]
```diff
@@ -646,13 +646,13 @@ It does not affect the notes' timeline and is not shuffled every time Random mod
...
+[Introduced in 22 February 2017 (2017-02-22) build \[b20170222.3 (Stable)\]](https://osu.ppy.sh/p/changelog?v=b20170222.3 "Release Notes for b20170222.3 (Stable)"), ScoreV2 mod uses the prototype ScoreV2 system instead of the current scoring system used by the mode which would be used in _osu!_ in the future to replace the current individual scoring system by modes. ScoreV2 system attempts to scale _all_ hit object scoring values to a hard limit of 1,000,000 (1 million) score limit at 1.00 score modifier excluding bonus spinner value of spinners in osu!standard mode, dendens in osu!taiko mode, and collected bananas (spinner equivalent) in osu!catch mode.
```
Comment (this1neguy) [inline wiki/Game_Modifiers/en.md]: * \+ 'the' before '22 February'
* remove "which [...] modes" in the first line. massive confusing run-on and the mode is explained in its own article anyway

File: wiki/Game_Modifiers/en.md [hunk 9]
```diff
@@ -156,7 +156,7 @@ Do note that the usage of No Fail mod reduces the amount of possible [Performanc
 
 > Shortcut key: `E`
 
-> Less zoom. ("Zoom" meaning here means how fast the beatmap's BPM is)
+> Less zoom. ("Zoom" referring to the beatmap's BPM)
```
Comment (TPGPL) [inline wiki/Game_Modifiers/en.md]: Zoom refers to the beatmap's BPM?

File: wiki/Game_Modifiers/en.md [hunk 10]
```diff
@@ -270,15 +270,15 @@ The method used to increase the speed does not increase the pitch of the song, b
...
+In [osu!taiko][osu!taiko wikilink] mode, since the hit leniency is greatly reduced on top of osu!taiko's already strict overall difficulty, along with slider ticks being considerably more difficult to hit (and reducing the value of dendens), Double Time is usually regarded as the hardest mod in osu!taiko and is seldom used.
 
 ---
 
 In [osu!catch][osu!catch wikilink] mode, as there is no overall difficulty to increase, this mod does nothing more than a BPM and approach rate multiplier and is thus only worth a 1.06x multiplier. Being the opposite of Half Time, though, Double Time greatly decreases the hyperdash leniency, making fruits requiring hyperdashes nigh impossible to catch in some cases.
 
 ---
 
-In [osu!mania][osu!mania wikilink] mode, the usage of Double Time mod will set the play as unranked play.
+In [osu!mania][osu!mania wikilink] mode, the usage of Double Time mod will result in an unranked play.
```
Comment (TPGPL) [inline wiki/Game_Modifiers/en.md]: This is outdated?
